packages/web/src/components/track/desktop/ConnectedTrackTile.tsx

```diff
@@ -236,7 +230,7 @@ const ConnectedTrackTile = ({
       includeRepost: hasStreamAccess,
       includeShare: true,
       includeTrackPage: true,
-      isArtistPick,
+      isArtistPick: false,
```

Comment:
for testing or do we want to always be false?

Reply:
great catch! will update